scripts/twister

```diff
@@ -292,6 +292,7 @@ Artificially long but functional example:
              "Default is to not do any tag filtering. Multiple invocations "
              "are treated as a logical 'or' relationship")
     parser.add_argument("-e", "--exclude-tag", action="append",
+                        default = ["stress"],
```

Comment:
no please, do not do it like this. mark the test as slow for now

Reply:
I have updated to use slow, thanks.